jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/Tabs.java

```diff
@@ -0,0 +1,100 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.TabsAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.interfaces.common.IsText;
+
+import java.util.List;
+
+public class Tabs extends UIBaseElement<TabsAssert> implements IsText {
+
+    /*TABS */
+    private WebList getTabs() {
+        return this.finds("[role='tab']");
+    }
+
+    private UIElement getTabByNumber(int tabNumber) {
+        return getTabs().get(tabNumber);
+    }
+
+    private UIElement getTabByText(String text) {
+        return getTabs().get(text);
+    }
+
+    private List<String> getTabsTitlesValues() {
+        return getTabs().values();
+    }
+
```

Comment:
In my opinion, all public methods of the element that you will use in tests should contain an annotation @JDIAction with corresponding description of.